lintly/parsers.py

```diff
@@ -199,6 +199,46 @@ def parse_violations(self, output):
         return violations
 
 
+class CfnLintParser(BaseLintParser):
+    """A parser for the `cfn-lint` command.
+
+      cfn-lint output example:
+
+      W2001 Parameter UnusedParameter not used.
+      template.yaml:2:9
+    """
+
+    def parse_violations(self, output):
+        violations = collections.defaultdict(list)
+
+        regex = re.compile(r"[EW]\d{4}\s")
+
+        next_line_is_path = False
+        current_violation = None
+        for line in output.strip().splitlines():
+            if regex.match(line):
+                # This line is a cfn-lint error or warning
+                next_line_is_path = True
+                current_violation = line
+            elif next_line_is_path:
+                # This line is a filepath:line_number:column_number
+                path, line_number, column = line.split(":")
+                path = self._normalize_path(path)
+
+                code, message = current_violation.split(" ", 1)
+
+                violation = Violation(line=int(line_number),
+                                      column=int(column),
+                                      code="`{}`".format(code),
```

Comment:
One last small thing -- let's also remove the backticks from the code. Take a look at the [other parsers](https://github.com/grantmcconnaughey/Lintly/blob/master/tests/test_parsers.py#L85) for examples of what those codes look like. For the cfn lint example I would expect the code to just be `"W2001"`.

Reply:
Back ticks are gone =)